README.md

```diff
@@ -12,6 +12,19 @@ The [`tinywallet`](./tinywallet) package contains a wallet based on the
 Each package may be installed from the [Python Package Index](https://pypi.org/)
 using the [`pip`](https://pip.pypa.io/) command as usual.
 
+## Requirements
+
+To run tinydecred on your machine you will need to set up the following. 
```

Comment:
You don't "run tinydecred". You can run **tinywallet**, or use the **decred** package.